sdk/loadtestservice/azure-developer-loadtesting/tests/test_smoke_async.py

```diff
@@ -36,30 +45,45 @@ async def test_smoke_create_or_update_test(self, loadtesting_endpoint):
         assert result is not None
 
     @LoadtestingPowerShellPreparer()
-    async def test_create_or_update_app_components(self, loadtesting_endpoint):
+    @recorded_by_proxy_async
+    async def test_create_or_update_app_components(
+            self,
+            loadtesting_endpoint,
+            loadtesting_test_id,
+            loadtesting_app_component,
+            loadtesting_subscription_id
+    ):
+        set_bodiless_matcher()
         client = self.create_client(endpoint=loadtesting_endpoint)
         result = await client.load_test_administration.create_or_update_app_components(
-            self.app_component,
+            loadtesting_app_component,
             {
                 "name": "app_component",
-                "testId": self.test_id,
+                "testId": loadtesting_test_id,
                 "value": {
-                    f"/subscriptions/{self.subscription_id}/resourceGroups/App-Service-Sample-Demo-rg/providers/Microsoft.Web/sites/App-Service-Sample-Demo": {
-                        "resourceId": f"/subscriptions/{self.subscription_id}/resourceGroups/App-Service-Sample-Demo-rg/providers/Microsoft.Web/sites/App-Service-Sample-Demo",
+                    f"/subscriptions/{loadtesting_subscription_id}/resourceGroups/App-Service-Sample-Demo-rg/providers/Microsoft.Web/sites/App-Service-Sample-Demo": {
+                        "resourceId": f"/subscriptions/{loadtesting_subscription_id}/resourceGroups/App-Service-Sample-Demo-rg/providers/Microsoft.Web/sites/App-Service-Sample-Demo",
                         "resourceName": "App-Service-Sample-Demo",
                         "resourceType": "Microsoft.Web/sites",
-                        "subscriptionId": self.subscription_id,
+                        "subscriptionId": loadtesting_subscription_id,
                     }
                 },
             },
         )
         assert result is not None
 
     @LoadtestingPowerShellPreparer()
-    async def test_get_app_components(self, loadtesting_endpoint):
+    @recorded_by_proxy_async
+    async def test_get_app_components(
+            self,
+            loadtesting_endpoint,
+            loadtesting_test_id,
+            loadtesting_app_component
+    ):
+        set_bodiless_matcher()
         client = self.create_client(endpoint=loadtesting_endpoint)
-        result = await client.load_test_administration.get_app_components(test_id=self.test_id)
+        result = await client.load_test_administration.get_app_components(test_id=loadtesting_test_id)
         assert result is not None
 
-        result = client.load_test_administration.get_app_components(name=self.app_component)
+        result = client.load_test_administration.get_app_components(name=loadtesting_app_component)
```

Comment:
the calls to async code should have `await` in them

Reply:
resolved